### DISCUSSION TITLE
problème coordonnées gps avec recepteur Hitec

### DISCUSSION THREAD

**Discussion by landrieve** [pascallanger/DIY-Multiprotocol-TX-Module#775] [Q&A] [answered]
bonjour,
j'utilise une tx16s toute neuve avec des récepteurs hitec. En branchant des gps sur ceux-ci je me suis aperçu que les coordonnées lut sur la radio étaient étonnées. Un peut décalées en nord et d'environ 800m vers l'est. je sais que les gps marche bien et je pense que c'est la traduction des coordonnées envoyées par la télémétrie qui n'est pas bonne. avez vous une solution.
merci

**Comment by pascallanger** [comment, marked as answer]
Bonsoir, le décodage de la télémétrie est effectué par OpenTX/EdgeTX. Merci de vous adresser à eux pour une résolution.
